src/pages/ReimbursementAccount/PersonalInfo/substeps/FullName.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import {View} from 'react-native';
+import lodashGet from 'lodash/get';
+import useLocalize from '../../../../hooks/useLocalize';
+import styles from '../../../../styles/styles';
+import Text from '../../../../components/Text';
+import TextInput from '../../../../components/TextInput';
+import CONST from '../../../../CONST';
+import Form from '../../../../components/Form';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import subStepPropTypes from '../../subStepPropTypes';
+import * as ValidationUtils from '../../../../libs/ValidationUtils';
+import {reimbursementAccountPropTypes} from '../../reimbursementAccountPropTypes';
+import HelpLinks from '../HelpLinks';
+import * as BankAccounts from '../../../../libs/actions/BankAccounts';
+
+const propTypes = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: reimbursementAccountPropTypes.isRequired,
+
+    ...subStepPropTypes,
+};
+
+const REQUIRED_FIELDS = [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.FIRST_NAME, CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.LAST_NAME];
+
+const validate = (values) => ValidationUtils.getFieldRequiredErrors(values, REQUIRED_FIELDS);
+
+function FullName({reimbursementAccount, onNext, isEditing}) {
+    const {translate} = useLocalize();
+
+    const defaultValues = {
+        firstName: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.FIRST_NAME], ''),
+        lastName: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.LAST_NAME], ''),
+    };
+
+    const handleSubmit = (values) => {
```

Comment:
What do you think to take this function out of `FullName` component, like

```
function updateOnyx(callback) {
   return (values) => {
        BankAccounts.updateOnyxVBBAData({
            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.FIRST_NAME]: values[CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.FIRST_NAME],
            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.LAST_NAME]: values[CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.LAST_NAME],
        });
        callback();
        }
   };
   ```
   
   later you can call it just
   ```
   updateOnyx(onNext);
   ```